_scripts/clone-legal-docs.sh

```diff
@@ -0,0 +1,155 @@
+            # one up
+            final_path="$(dirname "${final_path}")"
+        elif [ "${path_part}" != "." ]; then
+            final_path="${final_path}/${path_part}"
+        fi # otherwise ignore - current directory
+    done
+    IFS="${backup_ifs}"
+
+    echo "${final_path}"
+}
+# END: workaround for missing GNU realpath in BSD/MacOS-X
+
+DOWNLOAD_PATH="${FXA_LEGAL_DOCS_DOWNLOAD_PATH:-legal-docs}"
+SCRIPT_DIR="$(dirname "$0")"
+MODULE_PATH="$(get_realpath_missing "$SCRIPT_DIR/../packages/$MODULE")"
+
+[ -d "$MODULE_PATH" ] || error "The module/package does not exist at: $MODULE_PATH"
+
+# Clone and pull
+cd "$MODULE_PATH"
+
+if [ -n "${FXA_LEGAL_DOCS_COPY_FROM}" ]; then
+    echo "Using LEGAL_DOCS files from: ${FXA_LEGAL_DOCS_COPY_FROM}"
+    [ ! -d "${DOWNLOAD_PATH}" ] || rm -rf "${DOWNLOAD_PATH}"
+    ln -sf "${FXA_LEGAL_DOCS_COPY_FROM}" "${DOWNLOAD_PATH}"
+else
+    # Git repository to clone
+    FXA_LEGAL_DOCS_REPO="${FXA_LEGAL_DOCS_REPO:-https://github.com/mozilla/legal-docs.git}"
+
+    # branch name
+    FXA_LEGAL_DOCS_BRANCH="${FXA_LEGAL_DOCS_BRANCH:-main}"
+
+    # number of commits to truncate the history for specific commit checkout
+    FXA_LEGAL_DOCS_DEPTH="${FXA_LEGAL_DOCS_DEPTH:-1000}"
+
+    clone_opts=(--depth 1)
+
+    # fetch some history when required
+    [ -z "${FXA_LEGAL_DOCS_SHA}" ] || clone_opts=(--depth "${FXA_LEGAL_DOCS_DEPTH}")
+
+    # directly checkout to the desired branch and don't fetch others
+    [ -z "${FXA_LEGAL_DOCS_BRANCH}" ] || clone_opts+=(--branch "${FXA_LEGAL_DOCS_BRANCH}" --single-branch)
+
+    # Download LEGAL_DOCS using git
+    if [ ! -d "${DOWNLOAD_PATH}" ]; then
+        log "Downloading LEGAL_DOCS files from: ${FXA_LEGAL_DOCS_REPO}"
+        git clone "${clone_opts[@]}" "${FXA_LEGAL_DOCS_REPO}" "${DOWNLOAD_PATH}"
+    fi
+
+    log "Updating legal-docs files in: $(get_realpath_missing "${DOWNLOAD_PATH}")"
+
+    cd "${DOWNLOAD_PATH}"
+
+    # handle branch updates
+    if [ -n "${FXA_LEGAL_DOCS_BRANCH}" ]; then
+        git checkout --quiet -- .
+        git checkout --quiet --force "${FXA_LEGAL_DOCS_BRANCH}"
+        git pull --quiet origin "${FXA_LEGAL_DOCS_BRANCH}"
+    fi
+
+    # checkout to a specific commit
+    if [ -n "${FXA_LEGAL_DOCS_SHA}" ]; then
+        # ensure we have enough history
+        git fetch --quiet --depth "${FXA_LEGAL_DOCS_DEPTH}"
+        git checkout --quiet --force "${FXA_LEGAL_DOCS_SHA}"
+    fi
+    git rev-parse HEAD > git-head.txt
+fi
+
+copy_md() {
+    local md_name legal_docs_dir
+    md_name="$1"
+
+    log "Copying $md_name and parent locale directories into $MODULE_PATH/public/legal-docs/"
+
+    results=()
+    for src in **"/${md_name}.md"; do
+        [ -f "$src" ] || continue
+        legal_docs_dir="$MODULE_PATH/public/legal-docs/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
+        [ -d "$legal_docs_dir" ] || mkdir -p "$legal_docs_dir"
+        cp "$src" "$legal_docs_dir/$md_name.md"
```

Comment:
Mildly interesting (but not really) on how many locales only have translated one of the two files.

```sh
# in /legal-docs/ repo
git ls-files | grep -E "firefox_(privacy|cloud)_"

# ar/firefox_privacy_notice.md
# bn/firefox_privacy_notice.md
# cs/firefox_privacy_notice.md
# de/firefox_cloud_services_tos.md
# de/firefox_privacy_notice.md
# el/firefox_privacy_notice.md
# en/firefox_cloud_services_tos.md
# en/firefox_privacy_notice.md
# en_b/firefox_privacy_notice.md
# es-ES/firefox_cloud_services_tos.md
# es-ES/firefox_privacy_notice.md
# es-MX/firefox_privacy_notice.md
# fr/firefox_cloud_services_tos.md
# fr/firefox_privacy_notice.md
# hi/firefox_privacy_notice.md
# hr/firefox_privacy_notice.md
# hu/firefox_privacy_notice.md
# id/firefox_cloud_services_tos.md
# id/firefox_privacy_notice.md
# it/firefox_cloud_services_tos.md
# it/firefox_privacy_notice.md
# ja/firefox_cloud_services_tos.md
# ja/firefox_privacy_notice.md
# ko/firefox_privacy_notice.md
# mk/firefox_privacy_notice.md
# nl/firefox_cloud_services_tos.md
# nl/firefox_privacy_notice.md
# pl/firefox_cloud_services_tos.md
# pl/firefox_privacy_notice.md
# pt-BR/firefox_cloud_services_tos.md
# pt-BR/firefox_privacy_notice.md
# ro/firefox_privacy_notice.md
# ru/firefox_cloud_services_tos.md
# ru/firefox_privacy_notice.md
# sr/firefox_privacy_notice.md
# ta/firefox_privacy_notice.md
# tr/firefox_cloud_services_tos.md
# tr/firefox_privacy_notice.md
# vi/firefox_privacy_notice.md
# zh-CN/firefox_cloud_services_tos.md
# zh-CN/firefox_privacy_notice.md
# zh-TW/firefox_privacy_notice.md
```

```sh
git ls-files | grep -E "firefox_(privacy|cloud)" | grep "_privacy_" | wc -l # 29
git ls-files | grep -E "firefox_(privacy|cloud)" | grep "_cloud_" | wc -l # 13
```

Reply:
There are links available in the [legal-docs README](https://github.com/mozilla/legal-docs#localization) ;-) 

> Useful links:
> - A summary of the localization status is [available here](https://github.com/mozilla/legal-docs/tree/l10n_reference#readme), with a list of localized and non localized documents.
> - A full breakdown, with the list of locales for each document, is available in JSON format [here](https://github.com/mozilla/legal-docs/blob/l10n_reference/.github/stats.json).

Reply:
Nice, I'm going to make note of that list of locales in a follow up issue I'm filing here which will be helpful. 😁